From 2ba3e031847702049ecd99138a4ae7c2040ea5be Mon Sep 17 00:00:00 2001
From: Jan Wyszynski <jan.wyszynski@joyent.com>
Date: Wed, 14 Mar 2018 00:46:20 +0000
Subject: [PATCH] joyent/node-zkstream#32 ZKBuffer emits unhandled error on
 packet opcode CLOSE_SESSION Reviewed by: Alex Wilson <alex.wilson@joyent.com>
 Approved by: Alex Wilson <alex.wilson@joyent.com>

---
 lib/zk-buffer.js | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/zk-buffer.js b/lib/zk-buffer.js
index 51549f4..3556118 100644
--- a/lib/zk-buffer.js
+++ b/lib/zk-buffer.js
@@ -317,6 +317,7 @@ ZKBuffer.prototype.readResponse = function (xidMap) {
 		case 'PING':
 		case 'SYNC':
 		case 'DELETE':
+		case 'CLOSE_SESSION':
 			/*
 			 * No special response packet format -- error code in
 			 * header determines status of the request.
-- 
2.21.0

